src/utils/sendResponse.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const sendResponse = (res, resCode, resMessage, body) => {
+  res.statusCode = resCode;
+  res.statusMessage = resMessage;
+  res.end(body);
```

Comment:
Do it just here instead do it all times
```suggestion
  res.end(JSON.stringify(body));
```